docs/pages/x/api/date-pickers/index.md

```diff
@@ -55,6 +55,13 @@
 
 - [TimeClock](/x/api/date-pickers/time-clock/)
 
+### Toolbars
+
+- [DatePickerToolbar](/x/api/date-pickers/date-picker-toolbar/)
```

Comment:
Alternatively we could put it next to their picker (like for the tabs)

If we keep them here, could we keep the regular order "Date => Time => Date Time => Date Range" that we tend to use most of the time.